apps/remix-ide-e2e/src/tests/remixd.test.ts

```diff
@@ -71,6 +78,20 @@ module.exports = {
       .setSolidityCompilerVersion('soljson-v0.8.0+commit.c7dfd78e.js') // open-zeppelin moved to pragma ^0.8.0
       .testContracts('test_import_node_modules_with_github_import.sol', sources[4]['test_import_node_modules_with_github_import.sol'], ['ERC20', 'test11'])
   },
+  'Static Analysis run with remixd': function (browser) {
+    browser.testContracts('test_static_analysis_with_remixd_and_hardhat.sol', sources[5]['test_static_analysis_with_remixd_and_hardhat.sol'], ['test5'])
+      .clickLaunchIcon('solidityStaticAnalysis')
+      .click('#staticanalysisButton button')
+      .waitForElementPresent('#staticanalysisresult .warning', 2000, true, function () {
+        browser
+          .click('[data-id="staticAnalysisModuleMiscellaneous1"')
+          .getEditorValue((content) => {
```

Comment:
Does it also check if the required location is highlighted?

Reply:
this should be done now